### PR TITLE
Update Hive version in Docker README

### PR DESCRIPTION
**Pull request by mertyyanik**
### What changes were proposed in this pull request?
While the quickstart page for Docker was up to date, the readme was not. I added the updates on the page to the readme.
Quicstart page: https://hive.apache.org/developement/quickstart/

### Does this PR introduce _any_ user-facing change?
It allows the user to see the current readme page.

### Is the change a dependency upgrade?
No.

### How was this patch tested?
Since it only covers changes to the readme, I confirmed that the readme reads properly on markdown.

### CODE REVIEW

File: packaging/src/docker/README.md [hunk 1]
```diff
@@ -5,8 +5,39 @@ Run Apache Hive inside docker container in pseudo-distributed mode, provide the
 - Quick-start/Debugging/Prepare a test env for Hive
 
 
-### Quickstart
+## Quickstart
+### STEP 1: Pull the image
+- Pull the image from DockerHub: https://hub.docker.com/r/apache/hive/tags. 
 
+Here are the latest images:
+- 4.0.0-beta-1
```
Comment (deniskuzZ) [inline packaging/src/docker/README.md:13]: we should update with Hive 4.0.0, Hadoop 3.3.6, Tez 0.10.3

File: packaging/src/docker/README.md [hunk 2]
```diff
@@ -59,7 +60,7 @@ If the version is not provided, it will read the version from current `pom.xml`:
 
 For example, the following command uses Hive 3.1.3, Hadoop `hadoop.version` and Tez `tez.version` to build the image,
```
Comment (dengzhhu653) [inline packaging/src/docker/README.md:61]: Change `uses Hive 3.1.3` to `Hive 4.0.0`?

Comment (mertyyanik) [reply packaging/src/docker/README.md]: Done

**Review by dengzhhu653 — APPROVED** [review]

**Comment by dengzhhu653** [comment]
@mertyyanik Thank you for the PR, we use jira for issues/case management: https://cwiki.apache.org/confluence/display/Hive/HowToContribute#HowToContribute-JIRA, could you please create one and link this PR? Thanks in advance!

**Comment by mertyyanik** [comment]
Hello @dengzhhu653, I tried to open an account on Jira. Unfortunately, I did not receive an account opening email.

**Comment by dengzhhu653** [comment]
> Hello @dengzhhu653, I tried to open an account on Jira. Unfortunately, I did not receive an account opening email.

Sorry for the late reply, I just created a jira for this fix. Thank you for the PR!

**Pull request merged**